packages/harpparts/src/tuning/types/tuning-types.ts

```diff
@@ -7,6 +7,9 @@ export enum TuningIds {
   HarmonicMinor = 'Harmonic minor',
   MelodyMaker = 'Melody maker',
   EasyDiatonic = 'Easy diatonic',
+  TwelveHoleChromatic = 'Chromatic (12 hole)',
```

Comment:
Not really part of this feature, but for consistency these should be named ChromaticTwelveHole etc.

Reply:
But this isn't important, the naming isn't actually as consistent as I thought it was saving these and this is well out of scope for this little additive change.